games-board/cockatrice/cockatrice-20161030.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit cmake-utils eutils gnome2-utils
+
+DESCRIPTION="An open-source multiplatform software for playing card games over a network"
+HOMEPAGE="https://github.com/Cockatrice/Cockatrice"
+
+SRC_URI="https://github.com/Cockatrice/${PN}/archive/2016-10-30-Release.tar.gz -> ${P}.tar.gz"
+# As the default help/about display the sha1 we need it
+SHA1='3498b16'
+S=${WORKDIR}/"Cockatrice-2016-10-30-Release"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dedicated server"
```

Comment:
That's not quite what @SoapGentoo meant. `no-*` flags are very bad. You can enable flags by default using + instead.

```bash
USE="server +client +oracle"
```

What does the oracle switch actually do though?